contracts/LiquidityBridgeContract.sol

```diff
@@ -10,30 +10,34 @@ import './Bridge.sol';
 contract LiquidityBridgeContract {
 
     struct Quote {
+ 
         bytes20 fedBtcAddress;
         address lbcAddress;
         address liquidityProviderRskAddress;
         bytes btcRefundAddress;
         address rskRefundAddress;
         bytes liquidityProviderBtcAddress;
-        uint callFee;
-        uint penaltyFee;
+		// SDL: Versioning field is missing. Because no double-hash was used,
+		// then now the version field would need to go here I think
```

Comment:
By mistake, we didn't add a version field to the flyover protocol in the Bridge (Iris HF). Now we must find the best place to store it. The problem is that now, because the native Bridge arguments are a prefix of the full msg, we cannot distinguish old from new msgs by size. We can talk about this in a call